tests/test_cli.py

```diff
@@ -3,6 +3,7 @@
 import yaml
 
 CMD = 'python -m envars.envars'
+KKA = 'arn:aws:kms:eu-west-1:511042647617:key/fa5e3309-6af0-4df2-afd2-b21d480beaf4'
```

Comment:
What does KKA stand for?

Reply:
KMS KEY ARN, I should mock the boto3 call really

Reply:
mocking the call is proving a pain in the ****